app.js

```diff
@@ -76,8 +76,13 @@ io.on('connection', function(socket){
   socket.on('execute', function (formData) {
     generator.executeScript(socket, formData);
   });
+
   socket.on('deploy', function (data) {
-    ghdeploy.deployPages(socket, data);
+    deploy.deployPages(socket, data);
+  });
+
+  socket.on('heroku-deploy', function (formData) {
```

Comment:
If you're naming this `heroku-deploy`, don't leave the other as just `deploy`, name it `ghpages-deploy` or something similar. Or maybe `deploy/ghpages` and `deploy/heroku`??